src/axom/quest/IntersectionShaper.hpp

```diff
@@ -58,7 +58,7 @@
   #endif
 
   #if defined(AXOM_USE_HIP)
-    constexpr int HIP_BLOCK_SIZE = 256;
+    constexpr int HIP_BLOCK_SIZE = 64;
```

Comment:
This was the necessary change to get the `quest_intersection_shaper` unit test to not hang for rocm 5.2.3, amdclang++ for rocm 5.4.3 .

Hanging was occurring in the kernel that performs the `primal::clip` operation for a large iteration space. Failed consistently in the `outercircle.contour` example.

Reply:
That's strange. If the threadblock size is too large, the kernel shouldn't launch and report an error.  I don't think it should hang. Maybe the smaller block size makes it run faster?

Reply:
I also have tried running just this job for an hour with debug printf statements added, but that still seems to hang/no output from the kernel.

Sequential execution is fine.

Manually overriding the iteration space to a value <160,000 succeeded with a HIP execution policy.